lib/alexa.js

```diff
@@ -367,6 +367,9 @@ function ResponseBuilder(self) {
 
                 responseObject.response.directives = [audioPlayerDirective];
                 return this;
+            },
+            'setResponse': function () {
```

Comment:
My bad, it was specific to the way I was testing, the moment I upload to Lambda it worked... If you still want the docs, I'm will update those.